src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final int INDEX_OF_COUNT = 1;
```

Comment:
```suggestion
    private static final int INDEX_OF_COUNT = 1;
```
constants must have a private access modifier